### PR TITLE
Adds tech blog of hepsiburada.com which is an e-commerce website.

### PR DESCRIPTION
**Pull request by ilhan-mstf**

### CODE REVIEW

**Comment by ilhan-mstf** [comment]
Hi @ulgens, I really wonder that why you reacted with `thumbs down`. Can you provide explanatory feedback in order to improve the pull request since an emoji doesn't say too much about what else can be done.

**Comment by ulgens** [comment]
That emoji doesn't aim to provide any constructive criticism, that's not how reactions work, but of course i can explain what i think. In this list, we have Facebook, Google, Spotify, Instagram, Asana... and several others of the biggest technology companies we have today, but none of them are advertised as "biggest" or "best". Being `Turkey's biggest ecommerce website` is just about sale numbers, technology and engineering are a complete different category. This is not a place to seek that kind of attention.

TL; DR, I think the title is unpleasant and blog doesn't provide much value when you compare it with others on the list.

Btw, if you really need to know reason for a single thumbs-down emoji, please do not use internet, or at least don't share your thoughts. It will make you a happier person.

**Comment by ilhan-mstf** [comment]
Thank you for your feedback @ulgens, now, I got your point about the title of pull request. It actually describes the company since maintainers of the repo asked to describe blog briefly. Therefore, my intention is not to promote the company, my intention is to describe company. Before judging that it was a promotion, you can ask that whether it was a promotion or not. Furthermore, you can easily give feedback or warn me that the title can be understood as a `promotion` which I don't have in my mind.

Pressing a button is very easy. However, giving proper feedback requires effort and I believe that giving explanatory feedback is very important to improve things. We, as a society, always should look forward to improve our work. Therefore, I suggest rather than relying our judgement, we should --double-- check them that whether they are correct or not. Especially when analyzing things we should be objective not subjective and escape from our biases. To give example, maintainers of the repo said that 80% of the content should be technical and this requirement is not subjective. I thought that this blog satisfies this requirement.

Emoji is a very shallow version of communication, specifically discouraging ones. I suggest to be **careful** about giving feedback since it may yield people unproductive. We should support productivity and avoid discourage people.

I do not work for Hepsiburada (you can check it easily on my profile, I never worked for them and don't have such an intention). I just appreciated their work on medium and I wasn't aware of their blog. Therefore, I thought that many people may not aware of it too.

I am not agree with that their content does not provide value. It provides value since they talk about how to estimate time when using an unfamiliar technology stack, etc. Maybe it doesn't catch your attention but it may catch others, and others may learn something from that, again we shouldn't be **subjective**. Moreover, they don't promote their technology in their blog. However, I want to catch your attention that Google, Facebook, etc. continuously promote their technology on their blog. This is another topic and I don't want to enter its detail because it is my personal opinion.

If someone ask me to describe Hepsiburada, I will describe it as I described in this request. That doesn't mean that I promote them. I also think that Amazon is the biggest ecommerce website of USA. Does that mean that I promote Amazon? Or simple do Amazon need my promotion in this simple pull request? With similar approach, do Hepsiburada need my promotion in this simple pull request?

**Comment by ilhan-mstf** [comment]
It is reported that title of pull request can be understood as a promotion. Therefore, it is edited.

**Comment by ulgens** [comment]
I didn't read your comment my friend and i'm not planning to do. You're creating drama over a simple emoji, i'm not going to waste my time with this, sorry.

**Comment by ilhan-mstf** [comment]
Dramas are helpful if they are not real. Books, films, etc. are full of drama as they are aiming to explain something effectively and they do this intentionally.

I am sorry that my comments made you uncomfortable, but, I guess you are going to think twice next time. The real contribution of this pull request will be to gain you as a great feedback giver in the future.

**Comment by ilhan-mstf** [comment]
Dramas are helpful if they are not real. Books, films, etc. are full of drama as they are aiming to explain something effectively and they do this intentionally.

I am sorry that my comments made you(@ulgens ) uncomfortable, but, I guess you are going to think twice next time. The real contribution of this pull request will be to gain you as a great feedback giver in the future.

**Comment by kilimchoi** [comment]
You need more posts.
